src/Security/Security.Test/ScenarioTests/SecurityAlertTests.ps1

```diff
@@ -28,7 +28,7 @@ Get security alerts on a resource group and its resources
 #>
 function Get-AzureRmSecurityAlert-ResourceGroupScope
 {
-	$rgName = Get-TestResourceGroupName
```

Comment:
where was `Get-TestResourceGroupName` defined? why we do not need it anymore? (didn't see a removal of it)

Reply:
It is defined in Common.ps1
We use here Alerts Simulator resource group name instead